### PR TITLE
Qu auto/sorqa 82 fix

### PR DESCRIPTION
**Pull request by pk-sgent**
fixes

image

### CODE REVIEW

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactsLineListingSteps.java [hunk 1]
```diff
@@ -49,7 +49,8 @@ public ContactsLineListingSteps(
         "^I create a new Contact with specific data for DE version through Line Listing$",
         () -> {
           contactsLineListing = contactsLineListingService.buildGeneratedLineListingContactsDE();
-          selectDisease(contactsLineListing.getDisease());
+          //   selectDisease(contactsLineListing.getDisease()); // Disease should be set as a
+          // COVID-19 as default
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactsLineListingSteps.java:53]: we can remove commented code

File: sormas-e2e-tests/src/test/resources/features/sanity/web/LineListing.feature [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @UI @Sanity @Case
 Feature: Cases using Line Listing feature
 
-  @env_main @check
+  @env_main @check @testIt
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/LineListing.feature:4]: remove left tags

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java [hunk 3]
```diff
@@ -29,7 +30,7 @@ public CaseLineListingSteps(
         "^I create a new case in line listing feature popup for DE version$",
         () -> {
           caze = caseService.buildCaseForLineListingFeatureDE();
-          selectDisease(caze.getDisease());
+          // selectDisease(caze.getDisease()); // Disease should be set as a COVID-19 as default
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java:33]: we can remove commented code

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java [hunk 4]
```diff
@@ -85,7 +86,7 @@ public CaseLineListingSteps(
           webDriverHelpers.fillInWebElement(
               PERSON_ID_NAME_CONTACT_INFORMATION_LIKE_INPUT, caseName);
           webDriverHelpers.clickOnWebElementBySelector(CASE_APPLY_FILTERS_BUTTON);
-          webDriverHelpers.waitUntilNumberOfElementsIsReduceToGiven(CASE_DETAILED_TABLE_ROWS, 2);
+          TimeUnit.SECONDS.sleep(2); // wait for filter
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java:89]: isn't the next method covering the wait until filter is displayed?

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java (hunk 4 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseLineListingSteps.java:89]: Sleep works better in this case

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**